pkg/madmin/api-trace.go

```diff
@@ -0,0 +1,93 @@
+/*
+ * Minio Go Library for Amazon S3 Compatible Cloud Storage
```

Comment:
This is an incorrect license, it's not `Minio` Go library. Also, Minio should be MinIO